rcl/test/rcl/test_graph.cpp

```diff
@@ -852,6 +852,67 @@ check_graph_state(
   }
 }
 
+void
+check_entity_count(
```

Comment:
It looks to me like `check_entity_count` is completely replacing `check_graph_state`.  Given that `check_graph_state` isn't called anymore, we should remove it.

Reply:
> It looks to me like check_entity_count is completely replacing check_graph_state

almost..., but they are not exactly the same. 'check_entity_count' checks if number of entity is equal to expected number. `check_graph_state` checks if number of entity is equal to or greater than expected number.
I am okay to delete `check_graph_state`, but probably there would be some use cases for `check_graph_state` in the future.

Reply:
Well, it's completely unused as of this PR.  So I'd say we either add a test that uses it, or we delete it.  If we delete it, it will still be in the git history so we can always pull it back out if we need it in the future.

Reply:
sounds good to me!